### PR TITLE
:sparkles: Cypress testing claim with game

### PR DESCRIPTION
**Pull request by arvee-init**

### CODE REVIEW

File: cypress/integration/user-claimer/2-view-and-claim-w-game.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+				cy.contains( 'Play a game' )
+
+				// Click start game button
+				cy.contains( 'button', 'Start game' ).click()
+
+				// Expect score text
+				cy.contains( 'Score: 0 of' )
+
+				// Find buttons in page and click the one with h1 span value
+				cy.get( 'h1 span' ).invoke( 'text' ).then( (text) => { cy.contains( 'button', text ).click( {force: true} ) })
```
Comment (actuallymentor) [inline cypress/integration/user-claimer/2-view-and-claim-w-game.js:109]: Minor thing: try to keep to the overall style with the spaces. Ie in this case do `{ force: true }` with spaces.

File: cypress/integration/user-claimer/2-view-and-claim-w-game.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+				cy.wait(1000)
```
Comment (actuallymentor) [inline cypress/integration/user-claimer/2-view-and-claim-w-game.js:199]: These waits were optical right? If you no longer need them please remove to the CI can run as fast as it is able.

File: cypress/integration/user-claimer/2-view-and-claim-w-game.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+		cy.on('window:alert', response => {
+			expect( response ).to.contain( 'Deletion success' )
+		} )
+		cy.on('window:confirm', response => {
+			expect( response ).to.contain( 'Are you sure' )
+		} )
+
+		cy.contains( 'Delete QR dispenser' ).click()
+		cy.contains( 'Delete QR Dispenser' )
```
Comment (actuallymentor) [inline cypress/integration/user-claimer/2-view-and-claim-w-game.js:235]: This is a copy/paste duplicate I assume?

File: cypress/integration/user-claimer/2-view-and-claim-w-game.js [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+	const [ challenge_url ] = redirects
+	cy.log( `Redirect: `, challenge_url )
+	const [ base, challenge_redirect ] = challenge_url.split( '/#/claim/' )
+	const challenge = challenge_redirect.replace( '307: ' )
+	cy.log( `Challenge extracted: ${challenge}` )
+	return challenge
+
+}
+
+context( 'Claimer can view valid events with game', () => {
```
Comment (actuallymentor) [inline cypress/integration/user-claimer/2-view-and-claim-w-game.js:28]: Use something slightly more descriptive like "User can claim POAP after succeeding at challenge game"

**Review by actuallymentor — CHANGES_REQUESTED** [review]
Looks very good overall. Have a look at the minor edits.

If it takes less than ~hour see if you can add a test for "Game fails if user does not meet set score".

If that would take too long, leave it for now.

File: cypress/integration/user-claimer/2-view-and-claim-w-game.js (hunk 2 above)
Comment (arvee-init) [reply cypress/integration/user-claimer/2-view-and-claim-w-game.js]: The wait is not really a hold up as the game does need to run for a minimum of 10 sec. Removed it anyway.

**Pull request merged**